src/Pool.sol

```diff
@@ -372,9 +367,10 @@ abstract contract Pool {
     {
         (collected, dripped) = _collectInternal(senderAddr, currReceivers);
         Sender memory sender = senders[senderAddr];
-        StreamUpdates memory updates;
-        (withdrawn, updates) = _updateAnySender(
+        SenderId memory senderId = SenderId(false, senderAddr, 0);
```

Comment:
I think this format has higher readability 
```suggestion
        SenderId memory senderId = SenderId({isSubSender: false,...});
```

Reply:
Added helper functions, should be even more readable